### PR TITLE
doc: describe how to setup integration tests

### PR DESCRIPTION
**Pull request by coryan**
The integration tests need a small number of resources in the test
project, this document describes how to set them up and verify they
work.

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+These notes describe how to setup a project to run these integration tests.
+The main audience for these notes are developers working to improve the
+Functions Framework for C++. They assume the reader is familiar with GCP and
+with the Google Cloud SDK command-line tool.
+
+## Pre-requisites
+
+These notes assume you have setup a valid GCP project, with billing enabled.
+
+> :warning: most of these services will incur billing costs, probably in the
```
Comment (devbww) [inline ci/README.md:13]: s/most/Most/

File: ci/README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    --image="gcr.io/${GOOGLE_CLOUD_PROJECT}/gcf-hello-world-pubsub:latest" \
+    --region="us-central1" \
+    --platform="managed" \
+    --allow-unauthenticated
+```
+
+Create a Pub/Sub trigger for this deployment:
+
+```sh
+PROJECT_NUMBER="$(gcloud projects list --filter="PROJECT_ID=${GOOGLE_CLOUD_PROJECT}" --format="value(project_number)")"
```
Comment (devbww) [inline ci/README.md:183]: Perhaps split this long line?

File: ci/README.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+```sh
+gcloud logging read \
+    --project="${GOOGLE_CLOUD_PROJECT}" \
+    --format="value(textPayload)" \
+    "resource.type=cloud_run_revision AND resource.labels.service_name=gcf-hello-world-storage AND logName:stdout"
+```
+
+## Run the Cloud Build
+
+Finally verify this works by running the Cloud Build
```
Comment (devbww) [inline ci/README.md:249]: s/$/:/

**Review by devbww — APPROVED** [review]

File: ci/README.md (hunk 3 above)
Comment (coryan) [reply ci/README.md]: Done

File: ci/README.md (hunk 2 above)
Comment (coryan) [reply ci/README.md]: Done

File: ci/README.md (hunk 1 above)
Comment (coryan) [reply ci/README.md]: Fixed, thanks.

File: ci/README.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+### Grant the GCS service permissions to publish to Pub/Sub
+
+Get the GCS service account:
+
+```shell
+curl -X GET -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+  "https://storage.googleapis.com/storage/v1/projects/${GOOGLE_CLOUD_PROJECT}/serviceAccount"
+GCS_SA="... @gs-project-accounts.iam.gserviceaccount.com"
+```
```
Comment (devjgm) [inline ci/README.md:60]: Q: Is the `...` here indicating that we use some part of the output from the previous `curl` command?

**Review by devjgm — APPROVED** [review]

File: ci/README.md (hunk 4 above)
Comment (coryan) [reply ci/README.md:66]: Yes. Fixed the text around it to explain what is going on.

**Review by devjgm — APPROVED** [review]

**Pull request merged**